index.js

```diff
@@ -9,7 +10,14 @@ let headerMaxLength = 72
 // maximum allowed commit message length. If we didn't do this, the merged
 // commit would fail CI.
 if (process.env.TRAVIS_PULL_REQUEST && process.env.TRAVIS_PULL_REQUEST !== 'false') {
-  headerMaxLength -= 4 + process.env.TRAVIS_PULL_REQUEST.length
+  headerMaxLength -= prLength + process.env.TRAVIS_PULL_REQUEST.length
+} else {
+  // If Jenkins CI is building a GitHub PR using the Multibranch Pipeline,
+  // the CHANGE_ID env will contain the PR number. Similar approach to the
+  // Travis CI.
+  if (process.env.CHANGE_ID && process.env.CHANGE_ID !== 'false') {
+    headerMaxLength -= prLength + process.env.CHANGE_ID.length
+  }
```

Comment:
Nit: this will simplify the if-branching

```suggestion
} else if (process.env.CHANGE_ID && process.env.CHANGE_ID !== 'false') {
  // If Jenkins CI is building a GitHub PR using the Multibranch Pipeline,
  // the CHANGE_ID env will contain the PR number. Similar approach to the
  // Travis CI.
  headerMaxLength -= prLength + process.env.CHANGE_ID.length
```